experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -232,75 +232,79 @@ def test_invalid_experiment_requires_min_version_less_than_max_version(self):
             },
         )
 
-    def test_desktop_min_version_under_113_shows_warning(self):
-        experiment = NimbusExperimentFactory.create_with_lifecycle(
-            NimbusExperimentFactory.Lifecycles.CREATED,
-            firefox_min_version=NimbusExperiment.Version.FIREFOX_106,
-            is_rollout=True,
-            is_sticky=True,
-        )
-        for branch in experiment.treatment_branches:
-            branch.delete()
-        experiment.save()
-        serializer = NimbusReviewSerializer(
-            experiment,
-            data=NimbusReviewSerializer(
-                experiment,
-                context={"user": self.user},
-            ).data,
-            context={"user": self.user},
-        )
-        self.assertTrue(serializer.is_valid())
-        self.assertEqual(
-            serializer.warnings,
-            {
-                "firefox_min_version": [NimbusExperiment.ERROR_DESKTOP_ROLLOUT_VERSION],
-            },
-        )
-
-    def test_desktop_min_version_over_113_no_warning(self):
-        experiment = NimbusExperimentFactory.create_with_lifecycle(
-            NimbusExperimentFactory.Lifecycles.CREATED,
-            application=NimbusExperiment.Application.DESKTOP,
-            firefox_min_version=NimbusExperiment.Version.FIREFOX_118,
-            is_rollout=True,
-            is_sticky=True,
-        )
-        for branch in experiment.treatment_branches:
-            branch.delete()
-        experiment.save()
-        serializer = NimbusReviewSerializer(
-            experiment,
-            data=NimbusReviewSerializer(
+    def test_rollout_min_version_under_115_shows_warning(self):
```

Comment:
Would love some feedback on this test, context in code comments

Reply:
Instead of doing the `for` loop for the applications, you can pass them in using `parameterize.expand()` ([example](https://github.com/mozilla/experimenter/blob/41fe48a884c338ae073711d7f3e62b0a6767d848/experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py#L385))

Reply:
```py
@parameterized.expand(
        [
            (NimbusExperiment.Application.DESKTOP,),
            (NimbusExperiment.Application.FENIX,),
            (NimbusExperiment.Application.FOCUS_ANDROID,),
            (NimbusExperiment.Application.IOS,),
            (NimbusExperiment.Application.FOCUS_IOS,),
        ]
 )
def test_rollout_min_version_under_115_shows_warning(self, application):
     experiment = ...
         application=application,
         ...
```

Reply:
Magical, thank you!